server/src/test/java/io/pravega/schemaregistry/service/SchemaRegistryServiceTest.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.schemaregistry.service;
+
+import com.google.common.collect.Lists;
+import io.pravega.common.concurrent.Futures;
+import io.pravega.schemaregistry.ResultPage;
+import io.pravega.schemaregistry.contract.data.Compatibility;
+import io.pravega.schemaregistry.contract.data.GroupProperties;
+import io.pravega.schemaregistry.contract.data.SerializationFormat;
+import io.pravega.schemaregistry.storage.SchemaStore;
+import io.pravega.schemaregistry.storage.StoreExceptions;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.mock;
+
+public class SchemaRegistryServiceTest {
+    private SchemaRegistryService service;
+    private ScheduledExecutorService executor;
+
+    @Before
+    public void setup() {
+        executor = Executors.newScheduledThreadPool(5);
+    }
+    
+    @After
+    public void teardown() {
+        executor.shutdownNow();
```

Comment:
Please call io.pravega.common.concurrent.ExecutorServiceHelpers.shutdown(..) to cleanly shutdown the executor, which uses the 2-phase shutdown protocol described in [this ](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/ExecutorService.html#:~:text=Two%20different%20methods%20are%20provided,to%20stop%20currently%20executing%20tasks.)Oracle documentation.

Reply:
we dont need a two phase shutdown for the test code where this is the After method when the test has completed and it is ok to shutdown the executor Now as we dont want to wait for any processing to complete.